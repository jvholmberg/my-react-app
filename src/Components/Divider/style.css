.Divider {
  display: block;
  margin: 2.5rem auto;
  width: 35%;
  clear: both;
  border-bottom: .1rem solid #ececec;
}