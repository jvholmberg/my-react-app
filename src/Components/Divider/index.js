import React from 'react';

import './style.css';

/**
 * A Divider for marking the end of a section.
 * @returns {element} <Divider>
 */
const Divider = ({ className, children, ...props }) => {
  return (
    <span className='Divider'></span>
  );
};
export default Divider;
