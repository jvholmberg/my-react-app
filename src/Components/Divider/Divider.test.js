import React from 'react';

import Divider from './';

describe('Divider', () => {
  it('Empty matches snapshot', () => {
    expect(shallow(
      <Divider />
    )).toMatchSnapshot();
  });
});
