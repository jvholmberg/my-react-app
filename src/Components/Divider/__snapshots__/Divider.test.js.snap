// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider Empty matches snapshot 1`] = `
<span
  className="Divider"
/>
`;
